src/redux/reducers/index.ts

```diff
@@ -2,7 +2,10 @@ import { combineReducers } from 'redux'
 
 import employer from './employer'
 import jobseeker from './jobseeker'
+import skills from './skills'
+import skill from './skill'
 
-const rootReducer = () => combineReducers({ employer, jobseeker })
+const rootReducer = () =>
+  combineReducers({ employer, jobseeker, skills, skill })
```

Comment:
Perhaps change the name of the `skills` reducer ? I think the `skills` state that we fetch from server can be inside `reources` state:

`resources: { skills: [] }`

This way later on when we need to fetch more stuff from the api, we can categorize them as resource and put them in here as well